JS/see_article.js

```diff
@@ -172,8 +192,25 @@ function create_see_article(article) {
         }
         if (article_already_in_cart == false) {
             tab.push({ 'id': article.id, 'name': article.name, 'price': article.price, 'brand': article.brand, 'description': article.description, 'preview': article.preview, 'quantity': quantity_article });
+            document.getElementById("nb_article_in_cart").textContent = tab.length;
         }
         sessionStorage.setItem('articleToCart2', JSON.stringify(tab));
+        //Soit on affichera directement la pannier pour indiquer l'ajout soit on devra avoir un pop up
+       // location.href = "cart.html";
+
+        document.getElementById("overlay_alert").style.display = "block";
```

Comment:
hard coded CSS :)

Reply:
How would you do without using the css here?

Reply:
I would add a class called `hidden` which hold the display : None to the HTML node when I need to hide it and then I would just remove this class from his class list when I need the node the show up.

Reply:
Done.